requirements/edx/github.txt

```diff
@@ -99,4 +99,4 @@ git+https://github.com/open-craft/xblock-poll@v1.2.7#egg=xblock-poll==1.2.7
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.0.18#egg=xblock-drag-and-drop-v2==2.0.18
 git+https://github.com/open-craft/problem-builder.git@v2.10.3#egg=problem-builder==v2.10.3
 git+https://github.com/edx-solutions/xblock-image-explorer.git@v1.0.2#egg=xblock-image-explorer==v1.0.2
-git+https://github.com/raccoongang/xblock-pdf.git@v1.0.0#egg=xblock-pdf
+git+https://github.com/raccoongang/xblock-pdf.git@v1.0.0-rg#egg=xblock-pdf
```

Comment:
In one of another project, the same change works only without the `egg` tail. Have you tested this change locally?